scripts/reports/pr-report-github.js

```diff
@@ -0,0 +1,9 @@
+const {newClient} = require('./github-client');
+
+async function main() {
+  const c = newClient();
+  const issues = await c.issues.listForRepo({repo : 'ui-kit', owner: 'coveo', state: 'open'})
+  console.log(issues)
```

Comment:
What does this bot do? Is it just retrieving open issues and logging them to the console? Why is this helpful?